File: Front/eventos.client/src/app/models/palestrante.ts
import { Evento } from "./evento";
import { RedeSocial } from "./rede-social";

export interface Palestrante {
  id: number;
  nome: string;
  miniCurriculo: string;
  imagemURL: string;
  email: string;
  telefone: string;


  //FK
  redesSociais: RedeSocial[];
  palestrantesEventos: Evento[];
}

File: Back/Eventos.Domain/Models/Palestrante.cs
﻿namespace Eventos.Domain.Models;

public class Palestrante
{

    public int Id { get; set; }
    public string Nome { get; set; } = string.Empty;
    public string MiniCurriculo { get; set; } = string.Empty;
    public string ImagemURL { get; set; } = string.Empty;
    public string Telefone { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;

    //
    public IEnumerable<RedeSocial> RedesSociais { get; set; } 

    public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }

}
